src/cryptonote_core/uptime_proof.cpp

```diff
@@ -155,6 +160,7 @@ std::string Proof::bt_encode_uptime_proof(hf hardfork, cryptonote::network_type
         nettype == cryptonote::network_type::LOCALDEV) {
         proof.append("bk", tools::view_guts(pubkey_bls));
         proof.append("bp", tools::view_guts(pop_bls));
+        proof.append("gh", git_version);
```

Comment:
I don't see why we would only want this in ETH_TRANSITION but not beyond.

I do think, however, that it should only apply to non-mainnet nettype.